openmaptiles/tileset.py

```diff
@@ -181,7 +196,49 @@ def description(self) -> str:
 
     @property
     def buffer_size(self) -> int:
-        return self.definition['layer']['buffer_size']
+        """Each layer must have a default buffer size, with either `buffer_size` or `min_buffer_size` or both.
+        If both are set, buffer_size must be >= min_buffer_size.
+        min_buffer_size is only used when there is a global buffer size override,
+        e.g. if global is set to 0, and layer's min_buffer_size is set to 4, the result is 4.
+        Per layer tileset overrides are allowed for both buffer_size and min_buffer_size.
+        Per layer overrides have higher priority than global overrides, but less than ENV var.
```

Comment:
My understanding of this sentence is that env. variable has higher priority than per-layer min_buffer_size override. 

```suggestion
        Per layer `buffer_size` override has a higher priority than a global override, but less than ENV var.
        Per layer `min_buffer_size` tileset override has a higher priority than the layer's `min_buffer_size`.
        The resulting `min_buffer_size` is used if it is higher that the resulting `buffer_size`.
```

As far as I understand the code, the final buffer size of a layer is
```
max(
  first_of(
    TILE_BUFFER_SIZE environment variable,
    layer-specific override of the `buffer_size`,
    global override of the `buffer_size`,
    layer's `buffer_size`,
    layer's min_buffer_size,
    0),
  first_of(
    layer-specific override of min_buffer_size,
    global override of min_buffer_size,
    layer's min_buffer_size )
)
```